packages/react-scripts/config/webpack.config.js

```diff
@@ -254,6 +254,11 @@ module.exports = function (webpackEnv) {
       level: 'none',
     },
     optimization: {
+      splitChunks: isEnvDevelopment
+        ? undefined
+        : {
+            chunks: 'all',
+          },
```

Comment:
So this option can't be used in development mode anymore?
Is it webpack@5 issue, or specific for CRA only?

Reply:
If I recall, it caused issues with fast refresh.

Reply:
It would be good to add a comment about it. 
Forked repositories would benefit from this internal knowledge.